src/components/CommentForm.jsx

```diff
@@ -1,46 +1,50 @@
-import React, {useState} from 'react'
-import { postComment } from '../utils/api'
+import React, { useState } from 'react';
+import { postComment } from '../utils/api';
 
-const CommentForm = ({article_id, onCommentPosted}) => {
-    const [body, setBody] = useState('')
-    const [error, setError] = useState(null)
-    const [loading, setLoading] = useState(false)
-    const [success, setSuccess] = useState(false)
+const CommentForm = ({ article_id, username, onCommentPosted }) => {
+  const [body, setBody] = useState('');
```

Comment:
I don't like this naming convention here. 

Be as explicit as possible when naming state/variables